raycicmd/make.go

```diff
@@ -61,6 +62,20 @@ func parsePipelineFile(file string) (*pipelineGroup, error) {
 	return g, nil
 }
 
+func runTagFilterCommand(tagFilterCommand []string) ([]string, error) {
+	if len(tagFilterCommand) == 0 {
+		return nil, nil
+	}
```

Comment:
so using `nil` and empty slice to distinguish "run everything" and "run nothing" is quite error prone.

could you handle the "run everything" case in another way?

like, maybe return a struct here, with slice and also a boolean field like "all" to imply run everything?

also, could you add command existence check, so that we can have graceful fallback for older versions of ray?